### DISCUSSION TITLE
Migration to 2.1 failed because of multi-select picker error

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#8521] [Help] [answered]
I exported my app from Budibase 1.3.15 (local deployment) and imported it into 2.1.14 (cloud version), and my app started crashing with this browser console error:
![image](https://user-images.githubusercontent.com/93232582/199976243-f8bf6978-9870-45ee-b35e-9bf5c8e54114.png)

After some investigation, I was able to pinpoint the problem to one of my multi-select picker with a default value that uses Javascript:
![image](https://user-images.githubusercontent.com/93232582/199976582-5f65a6d5-bf53-4418-9da5-59bf1c11ad97.png)

Options source is "Custom", and it crashes no matter if I define options or not.
The very same picker works perfectly fine with BB 1.3.15. Could anyone help me out?

**Comment by melohagan** [comment, marked as answer]
Hey @dannyshin76 

Sorry to hear you're having issues. I was able to re-create and I've raised a bug here: https://github.com/Budibase/budibase/issues/8526

**Reply by dannyshin76** [reply]
Updated to 2.1.15 and my app works good now. Many thanks for your prompt action 🙏
